test/svelte.test.ts

```diff
@@ -19,9 +19,9 @@ test("svelte for loop", async ({ actual, expected }) => {
   expect(actual).toMatchInlineSnapshot(`
     {
       "branches": "0/0 (100%)",
-      "functions": "0/3 (0%)",
+      "functions": "0/4 (0%)",
```

Comment:
Similar to vue, `#each` is transformed to `$.each(..., ..., () => ..., () => ..., () => { ... })` and this last callback is now mapped correctly.

Reply:
This and the Vue test are here mostly to verify this library and Istanbul packages remap the same code similarly. In Svelte and Vue, I wouldn't necessarily call any of this "correct", but it's what their compilers output:

<img width="400"  src="https://github.com/user-attachments/assets/20260ef7-7087-4b66-8057-e92ccc0dbe99" />